frontend/src/components/Header/Header.style.ts

```diff
@@ -0,0 +1,64 @@
+import styled from '@emotion/styled';
+
+import { theme } from '@/style/theme';
+
+export const HeaderContainer = styled.nav`
+  position: fixed;
+  z-index: 1;
+  left: 0;
+
+  display: flex;
+  justify-content: center;
+
+  width: 100%;
+
+  background: white;
```

Comment:
지금은 상관없지만 나중에 대응 편하게 이런식으로 미리 넣어두어도 좋을 것 같아서 코멘트 남겨요~ 급한건 아니니까 그냥 살펴만 봐도 됩니당!
```suggestion
  background: theme.mode ==='dark' ? theme.color.dark.tertiary_900 : theme.color.light.white
```
```

Reply:
이러한 사항은 다크모드 디자인이 확정 나면 리팩토링 해볼게요!